static/js/admin/insert_update.js

```diff
@@ -85,7 +85,9 @@ define(
                     // If the input name does not match the regex, put the value in the post as usual
                     // _csrf is handled here
                     if ($this.attr('name').indexOf(fieldPrefix) != 0) {
-                        ajaxData[$this.attr('name')] = $this.val();
+                        if ($this.attr('type') != 'checkbox' || $this.is(":checked")) {
```

Comment:
:checkbox is not recommanded so i used jQuery recommandations instead.

http://api.jquery.com/checkbox-selector/